src/components/accession2/view/Accession2View.tsx

```diff
@@ -468,6 +468,12 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
             </Box>
           </Box>
         )}
+        {accession?.bagNumbers !== undefined && (
+          <Box display='flex' margin={isMobile ? themeObj.spacing(0, 0, 3, 6) : 0}>
+            <Typography fontWeight={600}> {strings.BAG_ID}: </Typography>
+            <Typography paddingLeft={1}> {accession.bagNumbers.join(', ')} </Typography>
```

Comment:
will this overflow to new lines if there are many bags with long names?

Reply:
Yes, it goes to new lines